package.json

```diff
@@ -33,9 +33,9 @@
     "content-disposition": "0.5.3",
     "content-type": "~1.0.4",
     "cookie": "0.4.0",
-    "cookie-signature": "1.0.6",
+    "cookie-signature": "1.1.0",
```

Comment:
This will create a confusing warning for users that says `cookie-signature@1.1.0: wanted: {"node":">=6.6.0"} (current: {"node":"1.8.4","npm":"2.9.0"})` but will also fail to install Express when they have strict engine set.

Reply:
Do you want me to remove the bump for `cookie-signature`, @dougwilson ? 🤔

Reply:
Yea, it will likely need to be held for 5.0